nvflare/private/fed/simulator/simulator_app_runner.py

```diff
@@ -25,7 +25,9 @@ def create_job_processing_context_properties(self, workspace, job_id):
 
 class SimulatorClientAppRunner(ClientAppRunner):
     def start_command_agent(self, args, client_runner, federated_client, fl_ctx):
-        pass
+        # pass
+        super().start_command_agent(args, client_runner, federated_client, fl_ctx)
+        self.command_agent.register_cell_cb()
```

Comment:
In the `ClientAppRunner`'s `start_command_agent` we call `self.command_agent.start(fl_ctx)`.

And in that `start` function we already have `self.register_cell_cb()`

So we don't need this method anymore?